server/index.js

```diff
@@ -16,16 +16,19 @@ async function createServer () {
 
   // Register the plugins
   await server.register(require('inert'))
-  await server.register(require('./plugins/views'))
-  await server.register(require('./plugins/router'))
-  await server.register(require('./plugins/error-pages'))
   await server.register(require('./plugins/api'))
+  await server.register(require('./plugins/error-pages'))
+  await server.register(require('./plugins/graceful-stop'))
+  await server.register(require('./plugins/views'))
```

Comment:
Good idea, I'll make a note to sort that out as a separate issue.

Reply:
I've tried this approach and hit strange errors in unit test runs with the 404 error test failing if run after other tests, but not when run individually. I think the cause is not correctly resetting the state of the `server` instance between tests but working this out is way beyond the scope of current activities so will have to wait.

Reply:
We now have a ticket in the ELM backlog (ELM-753) to look into this and fix whatever is wrong with the unit test setup/teardown so that refactoring the `server.register()` calls as suggestion will work.